app/routes.py

```diff
@@ -1,2 +1,289 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request
+import requests
+from datetime import datetime
+import os
+from dotenv import load_dotenv
 
+from tests.test_wave_06 import test_post_task_ids_to_goal
```

Comment:
These look like stray imports. Try to keep an eye out for these. VSCode can add them if we accept an autocomplete suggestion for a symbol found in another file.

Specifically, there's no need to import the dotenv stuff anywhere but in the main `__init__.py` file. Once the settings have been loaded at startup, they become available in the `os.environ` without any further action.

And in general, we wouldn't import something from a test module into our application. Test can depend on our application, but our application shouldn't depend on our tests.